### ISSUE TITLE
Implement `numOutputs` for `minimum`

### ISSUE THREAD

**Issue by nathantalewis** [tensorflow/haskell#150]
This seems very similar to #129.

The code in question:
```haskell
let y = (TF.minimum 0 (TF.expm1 x)) `TF.add` (TF.relu x)
```
The error message: `numOuputs not implemented for "Minimum"`

I'm using commit 79d8d7edea73e501719ae188c121821bf500d76f.

Alternatively, if there is an existing implementation of [`elu`](https://www.tensorflow.org/api_docs/python/tf/nn/elu) that I could use, how would I use it?

**Comment by fkm3** [comment]
Yeah, it is the same general problem, someone needs to add the gradient implementation for `minimum`. You will also hit the same issue for `expm1` I'd guess.

There is already an implementation for `maximum`. The `minimum` version should be exactly the same, but instead of `greaterEqual`, it should use `lessEqual` here https://github.com/tensorflow/haskell/blob/master/tensorflow-ops/src/TensorFlow/Gradient.hs#L540

**Comment by fkm3** [comment]
There is a builtin implementation of elu https://tensorflow.github.io/haskell/haddock/tensorflow-core-ops-0.1.0.0/TensorFlow-GenOps-Core.html#v:elu and its gradient function https://tensorflow.github.io/haskell/haddock/tensorflow-core-ops-0.1.0.0/TensorFlow-GenOps-Core.html#v:eluGrad, so that might be the easier route. (though you'd still need to plug that gradient function into the Gradient.hs framework)

**Comment by nathantalewis** [comment]
I must have missed the built-in version. Thanks for pointing it out. If I can get the gradient function plugged in I'll make a PR for it.
